docs/version0.48.md

```diff
@@ -42,6 +43,14 @@ From release 0.25.0 onwards, `zlibNX` hardware-accelerated data compression and
 
 For more information, see [`-XX:[+|-]UseZlibNX`](xxusezlibnx.md).
 
+### ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) JDWP support on Checkpoint/Restore In Userspace (CRIU) restore is enabled
+
+You can use the options that enable the JDWP support both on CRIU pre-checkpoint, and on restore as well.
+
+Also, a new parameter `suspendOnRestore`, specific to OpenJ9, is added to control the suspension of the target VM application on restore. You can use the `suspendOnRestore=n` setting to stop the suspension of the target application if the debugging is enabled on restore even if no debugging is set pre-checkpoint.
```

Comment:
The `suspendOnRestore=n` setting prevents the suspension of the target application if the debugging is enabled on restore regardless of whether the debugging was enabled pre-checkpoint or via the restore option file post-restore.